app/views/registrations/_subject_like_to_teach.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= f.govuk_radio_buttons_fieldset(:like_to_teach, legend: { text: 'Which subject would you like to teach if you return to teaching?' } ) do %>
-  <%= f.govuk_radio_button :like_to_teach, 'maths', label: { text: 'Maths' } %>
-  <%= f.govuk_radio_button :like_to_teach, 'physics', label: { text: 'Physics' } %>
-   <%= f.govuk_radio_button :like_to_teach, 'modern foreign language', label: { text: 'Modern foreign language' } %>
+  <%= f.govuk_radio_button :like_to_teach, 'a42655a1-2afa-e811-a981-000d3a276620', label: { text: 'Maths' } %>
```

Comment:
Feels like these should be constants somewhere

Reply:
Mapped to SubjectLikeToTeach::options["Maths"]..etc using ApiOptions module